src/fundus/publishers/de/focus.py

```diff
@@ -15,11 +15,11 @@
 
 class FocusParser(ParserProxy):
     class V1(BaseParser):
-        _paragraph_selector = CSSSelector("div.textBlock > p")
-        _summary_selector = CSSSelector("div.leadIn > p")
-        _subheadline_selector = CSSSelector("div.textBlock > h2")
+        _paragraph_selector = CSSSelector("div.textBlock > p , div[data-qa-article-content-text] > p")
+        _summary_selector = CSSSelector("div.leadIn > p, div.Article-Description ")
+        _subheadline_selector = CSSSelector("div.textBlock > h2, div[data-qa-article-content-text] > h2")
         _snippet_selector = XPath(
-            'string(//script[@type="text/javascript"][contains(text(), "window.bf__bfa_metadata")])'
+            'string(//script[@type="text/javascript"][contains(text(), "window.bf__bfa_metadata") or contains(text(), "pageAdKeyword")])'
```

Comment:
Is this still part of the new layout or a leftover of the former topic parsing fallback?

Reply:
No, that was also a leftover. It's fixed now